### PR TITLE
charges.json

### PR DESCRIPTION
**Pull request by GreenyMCR**

### CODE REVIEW

**Review by Ehbw — APPROVED** [review]
Manually corrected the format here, I recommend using a tool such as [jsonlint](https://jsonlint.com/) to validate and format your changes.

File: charges.json [hunk 1]
```diff
@@ -1,12 +1,242 @@
...
+        "details": "An individual who fails to adhere to a traffic control devices or move out the way upon the imediate approach of an authorized emergency vehicle making use of audible and visual signals",
+        "jailtime": 100,
+        "fine": 120,
+        "points": 3,
+        "category": "Moving Violations",
+        "color": "red"
+    },
+    { 
+        "id": 7,
+        "title": "Recklless Driving",
```
Comment (Ehbw) [inline charges.json:64]:
```suggestion
        "title": "Reckless Driving",
```

File: charges.json [hunk 2]
```diff
@@ -1,12 +1,242 @@
...
+        "fine": 2500,
+        "points": 3,
+        "category": "Moving Violations",
+        "color": "red"
+    },
+    { 
+        "id": 9,
+        "title": "Improper Lane Usage",
+        "details": "An individual who drives any vehicle in the left lane wherin he or she crosses a double (non-dotted) line going into oncoming lanes of traffic or fails to maintain a single lane",
+        "jailtime": ,
```
Comment (Ehbw) [inline charges.json:86]: Needs an jailtime or 0

File: charges.json [hunk 3]
```diff
@@ -1,12 +1,242 @@
...
+        "jailtime": 0,
+        "fine": 210,
+        "points": 0,
+        "category": "Moving Violations",
+        "color": "red"
+    },
+    { 
+        "id": 23,
+        "title": "Alchohol Bevrages In A Motor Vehicle",
+        "details": "No driver shall drink any alcoholic beverage while ina  vehicle upon a highway or roadway",
```
Comment (Ehbw) [inline charges.json:225]:
```suggestion
        "details": "No driver shall drink any alcoholic beverage while in a vehicle upon a highway or roadway",
```

**Review by Ehbw — CHANGES_REQUESTED** [review]

**Comment by GreenyMCR** [comment]
Updated

File: charges.json [hunk 4]
```diff
@@ -1,12 +1,693 @@
...
+        "jailtime": 180,
+        "fine": 2750,
+        "points": 0,
+        "category": "Crimes Against Property",
+        "color": "yellow"
     }
-]
\ No newline at end of file
+]
+]
```
Comment (Ehbw) [inline charges.json:693]: duplicate closing bracket
```suggestion

```

**Comment by Ehbw** [comment]
Just realised something, I never mentioned how the new jailtime works, every month is a minute, so a charge with 600 set as jailtime would be 600 irl minutes (reduced by doing tasks within prison), so you may want to re-do some of the jailtime, though its up to you.

**Pull request merged**